python/bgworker/background_process.py

```diff
@@ -91,10 +97,12 @@ def run(self):
                 self.log.info("Background worker process is running but should not run, stopping")
                 self.worker_stop()
 
-            import Queue
+            import queue
+            # even though we use multiprocessing.Queue, exceptions raised are
+            # from the standard queue module
             try:
                 item = self.q.get(timeout=1)
-            except Queue.Empty:
+            except queue.Empty:
```

Comment:
Hmm, this is problemetic with py2 compatibility. py3 calls it `queue` while py2 calls it `Queue` - should we check python version first and import by changing name?